src/pages/home/HomePage.tsx

```diff
@@ -38,7 +46,10 @@ export default class HomePage extends React.Component<IHomePageProps, {}>
     public render()
     {
         return (
-       		<QueryContainer/>
+        	<FlexCol style={{height: '100%'}}>
+				<a style={{alignSelf: 'center'}} onClick={() => this.setState({ mode: this.state.mode == 2 ? 1 : 2 })}>Switch to {this.state.mode == 2 ? 'old' : 'new'} view</a>
```

Comment:
Never?! That's amazing!   Not you, but other developers (like me) sometimes do make mistakes.  So for us, every js book in existence recommends === unless we specifically want type coercion.  That's the best practice.  I suppose you are welcome to ignore it?